std_msgs/Header header
geometry_msgs/Quaternion orientation
float32 thrust